server/src/main/java/org/elasticsearch/common/util/BytesRefArray.java

```diff
@@ -56,14 +56,14 @@ public BytesRefArray(StreamInput in, BigArrays bigArrays) throws IOException {
             // startOffsets
             size = in.readVLong();
             long sizeOfStartOffsets = size + 1;
-            startOffsets = bigArrays.newLongArray(sizeOfStartOffsets, true);
+            startOffsets = bigArrays.newLongArray(sizeOfStartOffsets, false);
```

Comment:
I have more of a question here, would not clearing of the arrays on resize cause any change of behaviour? I don't know enough about the existing uses of this API, perhaps it's all good, I just want to confirm that we've looked at all scenarios that use this code.

Reply:
clearing is not necessary, this is mistake I fix on the side, see line 39 and 41, it uses `false` for the normal instantiation.